src/main/resources/application.properties

```diff
@@ -8,4 +8,18 @@ spring.datasource.password=password
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
 spring.jpa.hibernate.ddl-auto=update
-spring.jpa.show-sql=true
\ No newline at end of file
+spring.jpa.show-sql=true
+
+#Kafka Consumer Settings
+server.port=9000
+spring.kafka.consumer.bootstrap-servers=localhost:9092
+spring.kafka.consumer.auto-offset-reset=earliest
+spring.kafka.consumer.group-id=group_id
+spring.kafka.consumer.key-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+spring.kafka.consumer.value-deserializer=org.apache.kafka.common.serialization.StringDeserializer
+
+#Kafka Producer Settings
+spring.kafka.producer.bootstrap-servers=localhost:9092
+spring.kafka.producer.key-serializer= org.apache.kafka.common.serialization.StringSerializer
+spring.kafka.producer.value-serializer=org.apache.kafka.common.serialization.StringSerializer
+
```

Comment:
I had to include this property to the branch for everything to work smoothly also. Here's the link: https://stackoverflow.com/questions/58789076/illegalstateexception-topics-xyz-is-are-not-present-and-missingtopicsfatal

`spring.kafka.listener.missing-topics-fatal=false`

Reply:
Added the dependency to the file. 
Oh yes forgot to mention. It's because I created a topic from command line and when the Springboot app is run it looks for a topic which in my case was Computer. Probably better to stop Kafka for searching the topic.